ansible/inventory.ini

```diff
@@ -0,0 +1,3 @@
+[myhosts]
```

Comment:
can we put the `ansible.cfg` and `inventory.ini` to root? then user could run `ansible` commands under project root dir directly rather than `cd ./ansible`

Reply:
BTW, please consider using the following `ansible.cfg`, specifically, `gathering=explicit` is highly encouraged, otherwise the ansible command may be dramatically slow
```
[defaults]
module_name=shell
gathering=explicit
```